data/json/items/comestibles/carnivore.json

```
@@ -932,5 +932,51 @@
     "material": [ "flesh" ],
     "volume": "250 ml",
     "fun": -30
+  },
+  {
+    "id": "meat_frond",
+    "copy-from": "meat_tainted",
+    "type": "COMESTIBLE",
+    "name": "alien fronds",
+    "use_action": "POISON",
+    "description": "The fleshy fronds harvested from an alien plant. The tiny membranous leaves attached and rigid and gut-like stem are certainly poisonous, and yet they have a paradoxically pleasant and inviting sweet smell.  Might be non-vegan.",
```

```suggestion
    "description": "The fleshy fronds harvested from an alien plant.  The tiny membranous leaves attached and rigid and gut-like stem are certainly poisonous, and yet they have a paradoxically pleasant and inviting sweet smell.  Might be non-vegan.",
```

While I understand the reason why the "...are certainly poisonous" part is in the description, but it seems kinda meta-gaming to me. How could player has known about this, especially if he didn't taste it before?

---

The other tainted pieces of meat warn you that eating them is a poor idea, but you are right, I'll reword it to sound less meta-gamey.